pyracf/common/utilities/logger.py

```diff
@@ -149,19 +149,33 @@ def __redact_request_dictionary(
     def __redact_string(
         self,
         input_string: Union[str, bytes],
-        start_ind: int,
-        end_pattern: Union[str, bytes],
-    ):
+        key: str,
```

Comment:
Change this to `trait_key` maybe for better clarity?

Reply:
sure! Done